airflow/contrib/operators/gcs_to_bq.py

```diff
@@ -43,9 +43,10 @@ class GoogleCloudStorageToBigQueryOperator(BaseOperator):
     :param source_objects: List of Google cloud storage URIs to load from. (templated)
         If source_format is 'DATASTORE_BACKUP', the list must only contain a single URI.
     :type source_objects: list[str]
-    :param destination_project_dataset_table: The dotted (<project>.)<dataset>.<table>
-        BigQuery table to load data into. If <project> is not included,
-        project will be the project defined in the connection json. (templated)
+    :param destination_project_dataset_table: The dotted
+        (<project>.|<project>:)<dataset>.<table> BigQuery table to load data into.
```

Comment:
```suggestion
        ``(<project>.|<project>:)<dataset>.<table>`` BigQuery table to load data into.
```
Could you fix this problem elsewhere?
I have a ready patch for you.
```bash
curl https://pastebin.com/raw/axcARnYH | git am
```

Reply:
Done @mik-laj